__DEFINES/setup.dm

```diff
@@ -967,12 +967,13 @@ var/list/RESTRICTED_CAMERA_NETWORKS = list( //Those networks can only be accesse
 
 #define RAD_ABSORB 2048
 #define REQUIRE_LIGHT 4096
+#define PLASMA_IMMUNE 8192
 
-#define CAN_BE_FAT 8192 // /vg/
+#define CAN_BE_FAT 16384 // /vg/
 
-#define IS_BULKY 16384 //can't wear exosuits, gloves, masks, or hardsuits
+#define IS_BULKY 32768 //can't wear exosuits, gloves, masks, or hardsuits
 
-#define NO_SKIN 32768
+#define NO_SKIN 65536 //Is there a limit to how high flag defines can go?
```

Comment:
okay yeah I just tested and it indeed doesn't work

I'll probably just replace the IS_WHITELISTED flag